### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by AleMdta**
In this milestone:
-"Works" section was added and styled using grid and flexbox.
-"About me" section was also added and styled with flexbox.
-"Read me" file was completed.
-Some images were added in the background to make it look like the design in Figma.

P.S. I really hope this project goes well. 
Thanks in advance.

### CODE REVIEW

**Review by frempongdev — COMMENTED** [review]
Hi @AleMdta , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❗ ❗ 


You have submitted a project, that does not have Linters set up correctly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AleMdta** [comment]
I fixed linters errors in this milestone.

File: style.css [hunk 1]
```diff
@@ -59,4 +68,204 @@
...
   margin-top: 50px;
   margin-bottom: 60px;
-}
\ No newline at end of file
+}
+
+.works {
+  margin-top: 0;
+  text-align: center;
+}
```
Comment (V-Blaze) [inline style.css]: > In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

- [ ] You have done great so far :+1: :+1: But there are styling issues and some requirements you need to make before moving to the next milestone, According to the project requirement you are mandated to use CSS GRID to lay out the works section, You didn't implement this in your CSS, I'll suggest you use this in your style.

File: style.css [hunk 2]
```diff
@@ -59,4 +68,204 @@
...
+  display: grid;
+  align-content: end;
+  width: 327px;
+  height: 474px;
+  border: 1px solid #d0d9d4;
+  background-color: #ebf0ee;
+  border-radius: 10px;
+  margin: 0 auto;
+  margin-bottom: 24px;
+}
```
Comment (V-Blaze) [inline style.css]: > Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

- [ ] Your work card looks like the given design but there are changes you need to improve on here.
- [ ] According to the requirements, you are meant to use Flexbox and not Grid to lay out the content of individual work cards, I'll suggest you use Flexbox as required :+1: :smile:

File: index.html [hunk 3]
```diff
@@ -42,6 +41,133 @@ <h1><span class="color-blue"> Hey there. <br> I'm Ale. </span><br><span id="titl
             </section>
             <a class="arrow" href="#"><img src="./images/Disabled.svg" alt=""></a>
         </section>
+        <section class="works">
+            <div class="title"> 
+                <h1 class="color-blue">My Recent Works</h1>
+            </div>
```
Comment (V-Blaze) [inline index.html]: - [ ] Following the best practices, you are meant to have only one `h1` tag on your page and that should be the main title, this is not the main title, and therefore ti should be in `2h or h3` tag, I'll suggest you update this accordingly. :+1: :smile:

File: index.html [hunk 4]
```diff
@@ -42,6 +41,133 @@ <h1><span class="color-blue"> Hey there. <br> I'm Ale. </span><br><span id="titl
...
+                            <li>HTML</li>
+                        </ul>
+                    </section>
+                    <button>See project</button>
+                </div>   
+            </section>
+        </section>
+        <section class="about-myself">
+            <div class="text">
+                <h1 class="color-blue">About me</h1>
```
Comment (V-Blaze) [inline index.html]: - [ ] Following the best practices, you are meant to have only one `h1` tag on your page and that should be the main title, this is not the main title, and therefore ti should be in `2h or h3` tag, I'll suggest you update this accordingly. :+1: :smile:

File: index.html [hunk 5]
```diff
@@ -42,6 +41,133 @@ <h1><span class="color-blue"> Hey there. <br> I'm Ale. </span><br><span id="titl
...
+            <section class="skills">
+                <h5 class="lang">Skills</h5>
+                <ul  class="lang-list">
+                    <li>Database Management</li>
+                    <li>Version Control</li>
+                    <li>CLI</li>
+                    <li>Web Development</li>
+                    <li>API Design</li>
+                </ul>
+            </section>
```
Comment (V-Blaze) [inline index.html]: - [ ] Aslo for this section, following the best practices, the category list should be in a ul tag with each item being an li, 
`
                        <ul class="languages">
                <li class="lang">Languages</li>
                    <ul  class="lang-list">
                        <li>JavaScript</li>
                        <li>Ruby</li>
                        <li>HTML</li>
                        <li>CSS</li>
                    </ul>
                <li class="lang">Frameworks</li>
                    <ul  class="lang-list">
                        <li>React.js</li>
                        <li>Ruby on Rails</li>
                        <li>RSpec</li>
                        <li>Capybara</li>
                        <li>Selenium</li>
                    </ul>
                <li class="lang">Skills</li>
                    <ul  class="lang-list">
                        <li>Database Management</li>
                        <li>Version Control</li>
                        <li>CLI</li>
                        <li>Web Development</li>
                        <li>API Design</li>
                    </ul>
            </ul>   
`

- - THis should be the structure of this section, I'll suggest you update this section. :+1: :100:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @AleMdta  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- UI looks great ✔
-  Great communication with reviewer👋 
-  Correct GitHub flow✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AleMdta** [comment]
In this milestone:
-"Works" section was styled using a combination of grid and flexbox.
-"About me" section was styled using flexbox.

**Review by Ofarouq310 — APPROVED** [review]
Hi @AleMdta, ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇 

giphy

## Highlights
✅ Followed GitHub flow.
✅ Descriptive PR title & summary.
✅ Meaningful commit messages.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**